packages/lib/modules/lbp/pool/usePriceInfo.ts

```diff
@@ -39,6 +57,37 @@ function toLbpPrices(apiPrices: LbpPriceChartDataFragment[]): LbpPrice[] {
   })
 }
 
+function aggregateToHourlyData(prices: LbpPriceChartDataFragment[]): HourlyDataPoint[] {
+  const hourlyDataMap = new Map<number, HourlyDataPoint>()
+
+  prices.forEach(price => {
+    const timestamp = secondsToMilliseconds(parseInt(price.timestamp.toString()))
```

Comment:
Jarek added `cumulativeFees` and `cumulativeVolume` to the API call (for calculating header stats). We probably don't need this aggregation.